showcase/src/app/demo/demo.module.ts

```diff
@@ -53,17 +58,21 @@ import {
 		ActionBarComponent,
 		ButtonComponent,
 		ExampleViewerComponent,
-		KeyboardDirectivesComponent,
 		PrettyPrintComponent,
+		DatePickerComponent,
+		ExampleViewerComponent,
+		KeyboardDirectivesComponent,
 		TableComponent
 	],
 	exports: [
 		ActionBarComponent,
 		ButtonComponent,
+		DatePickerComponent,
 		ExampleViewerComponent,
 		KeyboardDirectivesComponent,
 		PrettyPrintComponent,
 		TableComponent
-	]
+	],
+	providers: [{ provide: MAT_DATE_FORMATS, useValue: STARK_DATE_FORMATS }]
```

Comment:
Well it is for sure not trivial cause not intended to be used this way.
The regular way of changing the language is to set the dateAdapter: dateAdapter.setLocale("nl-be");
I checked on the moment documentation and when we format with LL, it displays the date in the set country fashion (here with LLLL):
EN: Friday, June 24, 2016 1:42 AM
DE: Freitag, 24. Juni 2016 01:42

So I think it will be fine :-)